doc/guides/security_release_process.md

```diff
@@ -1,61 +1,83 @@
 
-* [ ] Once agreement on the list and date has been agreed, validate that all
-  vulnerabilities have been assigned a CVE following the
-  [cve_management_process](https://github.com/nodejs/security-wg/blob/master/processes/cve_management_process.md).
+* [ ] Validate that all vulnerabilities have been assigned a CVE. Upstream deps
+  such as OpenSSL and NPM will have CVEs, issues reported on H1 may have CVEs,
+  otherwise allocate them by following the
+  [cve_management_process](https://github.com/nodejs/node/blob/master/doc/guides/cve_management_process.md).
 
 * [ ] Co-ordinate with the Release team members to line up one or more releasers
-  to do the releases on the agreed date.
+  to do the releases on the agreed date. Releaser: ***NAME of RELEASER(S)***
 
-* [ ] Prep for the pre-security announcement and final security announcement by
-  getting agreement on drafts following the
-  [security_announcement_process](https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md).
+* [ ] Prep for the security announcements by getting agreement on drafts (use
+  previously announced releases as the template):
+  - pre-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
+  - post-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
 
 ## Announcement (one week in advance of the planned release)
 
-* [ ] Ensure the pre-announce is sent out as outlined in the
-  [security_announcement_process](https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md).
+* Send pre-release announcement to
+  https://groups.google.com/forum/#!forum/nodejs-sec.  
+  One of the existing managers can give access (Ben
+  Noordhuis, Rod Vagg, Michael Dawson). ***LINK TO EMAIL***
+
+* Post pre-release announcement in vulnerabilities section of Nodejs.org blog
+  (https://github.com/nodejs/nodejs.org/tree/master/locale/en/blog/vulnerability).
+  Use last pre-release announcement as a template (it includes blog metadata
+  such as updates to the banner on the Node.js website to indicate security
+  releases are coming).  Submit PR and leave 1 hour for review. After one hour
+  even if no reviews, land anyway so that we don't have too big a gap between
+  post to nodejs-sec and blog. Text was already reviewed in security repo so is
+  unlikely to attract much additional comment. ***LINK TO BLOG PR AND POST***
 
 * [ ] Open an issue in the build working repository with a notification of the
   date for the security release.  Use this issue to co-ordinate with the build
   team to ensure there will be coverage/availability of build team resources the
   day of the release. Those who volunteer from the build WG should be available
-  in node-build during the release in case they are needed by the individual
-  doing the release.
-
-* [ ] Send an email to the docker official image
-  [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS)
```

Comment:
@tianon @yosifkit -- could you please subscribe to https://groups.google.com/forum/#!forum/nodejs-sec ? Having to send the announcement emails to two distribution lists seems unnecessary. Note that the list is SPAM free. The only posts to it are the pre and post release announcements, and the process currently requires docker-specific notifications both pre and post release.

Reply:
Absolutely, I'm subscribed and I believe @yosifkit is as well.

Reply:
So my extra email today was just spam :-(. Sorry! But it won't happen again if we get this landed.

Reply:
It's not a problem, I'd much rather get over-notified than under. 👍